### PR TITLE
Implement HTML output

### PR DESCRIPTION
**Pull request by sundowndev**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | yes
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #368
| ❓ Documentation  | yes 

## Description

This PR introduces a new output, allowing users to better understand the scan result when the Terminal is just not enough.

## Example

```
$ driftctl scan -o html://output.html
```

### CODE REVIEW

File: pkg/cmd/scan/output/assets/index.vue.tmpl [hunk 1]
```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
```
Comment (sundowndev) [inline pkg/cmd/scan/output/assets/index.vue.tmpl:1]: unused file to delete

File: pkg/cmd/scan/output/assets/vue.global.prod.js [hunk 2]
```diff
@@ -0,0 +1 @@
...e?n.recurseCount++:t.push({vnode:e,recurseCount:0});const o=e.component&&e.component.parent;e=o&&o.vnode}return t}();if(o)St(o,n,11,[e+t.join(""),n&&n.proxy,r.map((({vnode:e})=>`at <${Ir(n,e.type)}>`)).join("\n"),r]);else{const n=[`[Vue warn]: ${e}`,...t];r.length&&n.push("\n",...function(e){const t=[];return e.forEach(((e,n)=>{t.push(...0===n?[]:["\n"],...function({vnode:e,recurseCount:t}){const n=t>0?`... (${t} recursive calls)`:"",o=` at <${Ir(e.component,e.type,!!e.component&&null==e.component.parent)}`,r=">"+n;return e.props?[o,..._t(e.props),r]:[o+r]}(e))})),t}(r)),console.warn(...n)}ae()},e.watch=Bn,e.watchEffect=In,e.withCtx=nn,e.withDirectives=function(e,t){if(null===Yt)return e;const n=Yt.proxy,o=e.dirs||(e.dirs=[]);for(let r=0;r<t.length;r++){let[e,s,i,l=m]=t[r];F(e)&&(e={mounted:e,updated:e}),o.push({dir:e,instance:n,value:s,oldValue:void 0,arg:i,modifiers:l})}return e},e.withKeys=(e,t)=>n=>{if(!("key"in n))return;const o=z(n.key);return t.some((e=>e===o||Us[e]===o))?e(n):void 0},e.withModifiers=(e,t)=>(n,...o)=>{for(let e=0;e<t.length;e++){const o=js[t[e]];if(o&&o(n,t))return}return e(n,...o)},e.withScopeId=e=>nn,Object.defineProperty(e,"__esModule",{value:!0}),e}({});
```
Comment (sundowndev) [inline pkg/cmd/scan/output/assets/vue.global.prod.js:1]: unused file to delete

File: pkg/cmd/scan/output/console.go [hunk 3]
```diff
@@ -168,7 +168,7 @@ func (c Console) writeSummary(analysis *analyser.Analysis) {
 func prettify(resource interface{}) string {
 	res := reflect.ValueOf(resource)
 	if resource == nil || res.Kind() == reflect.Ptr && res.IsNil() {
-		return "<nil>"
+		return "<null>"
```
Comment (wbeuil) [inline pkg/cmd/scan/output/console.go:171]: Why did you change the wording here ?

**Review by wbeuil — COMMENTED** [review]
Did you change something for the number of alerts as per our call yesterday ?

File: pkg/cmd/scan/output/html.go [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+		}
+	}
+
+	return types
+}
+
+func htmlPrettify(resource interface{}) string {
+	res := reflect.ValueOf(resource)
+	if resource == nil || res.Kind() == reflect.Ptr && res.IsNil() {
+		return "&lt;null&gt;"
```
Comment (wbeuil) [inline pkg/cmd/scan/output/html.go:173]: Just return `null`

File: pkg/cmd/scan/output/console.go (hunk 3 above)
Comment (sundowndev) [reply pkg/cmd/scan/output/console.go]: I'll revert this change as it's not related to this PR (this function was duplicated). I'll create another PR to rewrite it to null.

**Review by wbeuil — APPROVED** [review]
LGTM, nice job !
